bindata/scripts/enable-rdma.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+REDHAT_RELEASE_FILE="/etc/redhat-release"
+
+function kmod_isloaded {
+  if grep --quiet '\(^ib\|^rdma\)' <(chroot /host/ lsmod); then
```

Comment:
nit: can we list all the module names instead of using regex?

Reply:
The loaded module names could be different with different hardware. This line only tends to check if any of the rdma kernel modules has been loaded, if any of the kmod is loaded, that means the rdma service has been activated.